src/components/SidePanel/SidePanel.tsx

```diff
@@ -156,6 +168,8 @@ class SidePanel extends React.Component<ISidePanelProps, ISidePanelState, {}> {
 		preventBodyScroll: false,
 		topOffset: 0,
 		width: 240,
+		minWidth: -Infinity,
+		maxWidth: window.innerWidth,
```

Comment:
This leaves one final edge case that I can think of. The default `maxWidth` will be pinned to the `maxWidth` when the component mounts (or when the module loads, not sure off the top of my head) which would mean that when the user resizes their browser it wouldn't be taken into account.

Perhaps a better approach would be to set this to `Infinity` and then in the `handleResize` handler check if `this.props.maxWidth === Infinity` and read directly from `window.innerWidth` there. I _think_ that would avoid that edge case.